index.html

```diff
@@ -0,0 +1,202 @@
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <br>
+        <br>
+          <a href="https://www.facebook.com/ndonyi.maurice" class="resume">See Project</a>
+        </div>
+    </div>
+   
+  </div>
+<div class="project1">
+    <div class="project_1st">
+        <div class="one">
+            <img src="./Images/Image_4.png" alt="">
+        </div>
+      <div class="detail">
+        <h2>Multi-Post
+          Stories</h2>
+        <ul>
+            <li><a href="#">CANOPY</a></li>
+            <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+            <li><a href="#">2015</a></li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="all">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <br>
+        <br>
+          <a href="https://www.facebook.com/ndonyi.maurice" class="resume">See Project</a>
+        </div>
+    </div>
+   
+  </div>
+<div class="project2">
+  <div class="project_1st">
+      <div class="two">
+          <img src="./Images/IMAGE5.PNG.png" alt="">
+      </div>
+    <div class="detail">
+      <h2>Tonic</h2>
+      <ul>
+          <li><a href="#">CANOPY</a></li>
+          <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+          <li><a href="#">Back End Dev</a></li>
+          <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+          <li><a href="#">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="all">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JAVASCRIPT</li>
+        </ul>
+        <br>
+        <br>
+        <a href="https://www.facebook.com/ndonyi.maurice" class="resume">See Project</a>
+      </div>
+  </div>
+ 
+</div>
+<div class="project3">
+  <div class="project_1st">
+      <div class="three">
+          <img src="./Images/Image_2.png" alt="">
+      </div>
+    <div class="detail">
+      <h2>Multi-Post
+        Stories</h2>
+      <ul>
+          <li><a href="#">CANOPY</a></li>
+          <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+          <li><a href="#">Back End Dev</a></li>
+          <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+          <li><a href="#">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="all">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JAVASCRIPT</li>
+        </ul>
+        <br>
+        <br>
+        <a href="https://github.com/citec-47/PORTFOLIO-TWO/blob/set-up/README.md" class="resume">See Project</a>
+      </div>
+  </div>
+ </div>
+ </section>
+<section class="about_me">
```

Comment:
Please consider reconsidering your design to match the Figma template. It includes similar margins in this case etc. Moreover, on the right side of Languages, there needs to be an icon that is not showing consider rechecking it.